core/drivers/gic.c

```diff
@@ -948,11 +950,34 @@ static void __maybe_unused gic_native_itr_handler(void)
 	iar = gic_read_iar(gd);
 	id = iar & GICC_IAR_IT_ID_MASK;
 
-	if (id <= gd->max_it)
+	if (id <= gd->max_it) {
 		interrupt_call_handlers(&gd->chip, id);
-	else
-		DMSG("ignoring interrupt %" PRIu32, id);
+		goto write_eoir;
+	} else if (IS_ENABLED(CFG_ARM_GICV3) && id >= 1020 && id <= 1023) {
+		/*
+		 * Special INTIDs
+		 * 1020: Interrupt expected to be handled at SEL1 or SEL2
```

Comment:
It might be worth mentioning that it's only received at `EL3`, if I understood correctly.